src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/WorkflowTests/ChangeNotification.cs

```diff
@@ -0,0 +1,323 @@
+            await ConfirmApprenticeship(apprenticeship, new ConfirmationBuilder().AsIncomplete());
+
+            var retrieved = await GetApprenticeship(apprenticeship);
+            retrieved.Should().BeEquivalentTo(new
+            {
+                DisplayChangeNotification = false,
+            });
+        }
+
+        private async Task ConfirmApprenticeship(ApprenticeshipDto apprenticeship, ConfirmationBuilder confirm)
+        {
+            context.Time.Now = context.Time.Now.AddDays(1);
+
+            apprenticeship = await GetApprenticeship(apprenticeship);
+
+            foreach (var payload in confirm.BuildAll())
+            {
+                var r4 = await client.PostValueAsync(
+                    $"apprentices/{apprenticeship.ApprenticeId}/apprenticeships/{apprenticeship.Id}/statements/{apprenticeship.CommitmentStatementId}/{payload.Item1}",
+                    payload.Item2);
+                r4.EnsureSuccessStatusCode();
+            }
+        }
+
+        private async Task ChangeApprenticeship(ChangeBuilder change)
+        {
+            context.Time.Now = context.Time.Now.AddDays(1);
+            var data = change.ChangedOn(context.Time.Now).Build();
+            var r1 = await client.PostValueAsync("apprenticeships/change", data);
+            r1.EnsureSuccessStatusCode();
+        }
+
+        private async Task<ApprenticeshipDto> GetApprenticeship(ApprenticeshipDto apprenticeship)
+        {
+            var (r2, apprenticeships) = await client.GetValueAsync<List<ApprenticeshipDto>>(
+                $"apprentices/{apprenticeship.ApprenticeId}/apprenticeships");
+            r2.EnsureSuccessStatusCode();
+
+            apprenticeships.Should().NotBeEmpty();
+            return apprenticeships.Last(); ;
+        }
+
+        public async Task<(ApprenticeshipDto, DateTime approvedOn)> CreateApprenticeship(HttpClient client)
+        {
+            var create = fixture.Build<CreateRegistrationCommand>()
+                .With(p => p.Email, (MailAddress adr) => adr.ToString())
+                .Create();
+
+            var createResponse = await client.PostValueAsync("apprenticeships", create);
+            createResponse.EnsureSuccessStatusCode();
+
+            var verify = fixture.Build<VerifyRegistrationCommand>()
+                .With(x => x.ApprenticeId, create.ApprenticeId)
+                .With(p => p.Email, create.Email)
+                .Create();
+
+            var verifyResponse = await client.PostValueAsync("registrations", verify);
+            verifyResponse.EnsureSuccessStatusCode();
+
+            var (getResponse, apprenticeships) = await client.GetValueAsync<List<ApprenticeshipDto>>($"apprentices/{create.ApprenticeId}/apprenticeships");
+            getResponse.EnsureSuccessStatusCode();
+
+            context.Time.Now = create.CommitmentsApprovedOn;
+
+            return (apprenticeships[0], create.CommitmentsApprovedOn);
+        }
```

Comment:
I'm not a big fan of this pattern, because the test is in reality spanning the entire flow of the system, and not just the points we are interested in (for this test case). Personally I think it belongs to the Acceptance Tests written by the our Tester. 

But